ui/pages/notifications/notifications.tsx

```diff
@@ -162,7 +161,7 @@ export default function Notifications() {
   const history = useHistory();
   const t = useI18nContext();
 
-  useEffectDeleteExpiredNotifications();
+  useEffectDeleteExpiredSnapNotifications();
```

Comment:
It's unclear for me why we need this and `useSnapNotificationTimeouts`?

Reply:
Good catch, forgot to delete this.